### DISCUSSION TITLE
hci_usb device recognition fails on the nucleo_wb55rg board.

### DISCUSSION THREAD

**Discussion by 2songh** [zephyrproject-rtos/zephyr#57372] [Q&A]
**Is your feature request related to a problem? Please describe.**
Route: zephyr/samples/bluetooth/hci_usb
Board: nucleo_wb55rg
I built this sample program and flashed it to the board.

The device is recognized normally but is marked as unavailable.
- **This device cannot be started. (Code 10)**
- **This is an unsupported request.**

**Describe the solution you'd like**
I want to use it as a bluetooth dongle and test it.

**Additional context**
![image](https://user-images.githubusercontent.com/126455296/234732193-e9706acc-1e33-4ce1-b6c8-f38676a18c51.png)
Is this nucleo_wb55rg board not supported?

**Comment by nordicjm** [comment]
This is expected behaviour on windows as it does not support devices bluetooth LE only radios and requires classic to be at least supported.

**Comment by 2songh** [comment]
> This is expected behaviour on windows as it does not support devices bluetooth LE only radios and requires classic to be at least supported.

Are you saying that this function is not available for Windows?
Or does the board itself not support USB dongle function?

**Comment by nordicjm** [comment]
A windows issue, try the device on linux. @jfischer-no can't seem to find the issue where this was discovered with bluetooth LE only devices not working on windows, do you remember where it was?

**Comment by 2songh** [comment]
> A windows issue, try the device on linux. @jfischer-no can't seem to find the issue where this was discovered with bluetooth LE only devices not working on windows, do you remember where it was?


I don't know about that.
Windows seems to have a lot of bad issues.

Any solution to this problem?
Is it only available on Linux?

**Comment by nordicjm** [comment]
Here it is: https://github.com/zephyrproject-rtos/zephyr/issues/23866

**Comment by 2songh** [comment]
> Here it is: #23866

Thank you for answer.
It was a great help in understanding the problem.

**Comment by 2songh** [comment]
> Here it is: #23866

Sorry for reopening the issue.
I have additional questions.
I tested the USB dongle sample program on Ubuntu (v22.04).

1. The device name is displayed as the set name.
![캡처2](https://user-images.githubusercontent.com/126455296/235088697-f4979e57-b93b-4444-aed9-7d39048a0b33.PNG)

2. When the device is connected, the Bluetooth setting changes to ON.

3.  Devices capable of Bluetooth connection are displayed.
![캡처3](https://user-images.githubusercontent.com/126455296/235089920-5485b6e4-84f0-41d2-8b61-d4f98257d3f1.PNG)

4. When I click the device I want to connect to, the connection is not made.
    (Other Bluetooth dongles are connected normally)

**Question**
Is nucleo_wb55rg board problem?
Or is the function not yet complete in Zephyr?

I want this sample program function to work normally on my board.

**Comment by nordicjm** [comment]
Would suggest using `bluetoothctl` from the command line instead to see if it works

**Comment by 2songh** [comment]
> `bluetoothctl`작동하는지 확인하기 위해 대신 명령줄에서 사용하는 것이 좋습니다.

![image](https://user-images.githubusercontent.com/126455296/235094195-ae12fa07-4457-4d2a-81ed-7e67c6c218c0.png)

I used 'bluetoothctl'.
The action seems to be working normally.

It works the same as other USB dongles, but the connection is not made.

**Comment by jfischer-no** [comment]
>Sorry for reopening the issue.
I have additional questions.

That is okay, moved to discussions.

**Reply by 2songh** [reply]
Thank you for your interest in my issue.
I'm curious what could be causing that problem.

Is the board causing the problem?
Or is there something wrong with my testing process?

I only have nucleo_wb55rg board.
I wonder if other boards normally function as Bluetooth dongles.

**Comment by 2songh** [comment]
Can I get some answers on the matter?

Is the nucleo_wb55rg  board the cause?
If I use another board, will it work normally?

If the nucleo_wb55rg board is the cause of the problem, I'll close the discussion and find another way.

**Reply by erwango** [reply]
Please raise a github issue providing all details required to analyze the issue.

**Reply by 2songh** [reply]
I already raised an issue.
https://github.com/zephyrproject-rtos/zephyr/issues/57315

Do you need additional information?

**Comment by jfischer-no** [comment]
Maybe @erwango can help you with nucleo_wb55rg
